pages/docs/reference/compatibility-guide-14.md

```diff
@@ -687,3 +689,36 @@ Remember that those definitions are given only for pure Kotlin. Compatibility of
 > - < 1.4: digraphs are capitalized in the upper case (`Ǆ`)
 > - \>= 1.4: digraphs are capitalized in the title case (`ǅ`)
 
+## Tools
+
+### Compiler arguments with delimiter characters must be passed in double quotes on Windows
+
+> **Issue**: [KT-30211](https://youtrack.jetbrains.com/issue/KT-30211)
+> 
+> **Component**: CLI utils
+> 
+> **Incompatible change type**: behavioral
```

Comment:
I think we can leave out these for tools, as probably any change will be behavioral in some sense...